resources/views/partials/example-requests/python.blade.php

```diff
@@ -0,0 +1,27 @@
+```python
+import requests
+import json
+
+url = '{{ trim(config('app.docs_url') ?: config('app.url'), '/')}}/{{ ltrim($route['uri'], '/') }}'
```

Comment:
Please check out the recent changes to the other language templates. Should be using `apidoc.base_url`